dask/dataframe/multi.py

```diff
@@ -466,6 +711,7 @@ def merge(
     npartitions=None,
     shuffle=None,
     max_branch=None,
+    bcast=None,
```

Comment:
I have a slight preference for an unabbreviated name if possible.  Maybe `broadcast=` instead?

Reply:
`broadcast=` makes sense to me